charts/search-index-env-sync/snapshot.sh

```diff
@@ -52,9 +52,14 @@ snapshot_valid () {
 
 restore () {
   snapshot_valid
-  $curl -XDELETE "$ES_URL/_all"
   local result
-  result=$($curl -XPOST "$ES_URL/_snapshot/$SNAPSHOT_REPO/$SNAPSHOT_NAME/_restore")
+  result=$($curl -XDELETE "$ES_URL/*,-.*")  # Delete all except system indices.
```

Comment:
What does setting this output to results do if it's getting overwritten by the restore curl?